docker/Dockerfile.base

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+  apt-get -y install \
+    htop \
+    unzip bzip2 \
+    wget curl \
+    emacs25-nox \
+    default-jdk \
+    python3 python3-pip && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN python3 -m pip install -U \
+  pip decorator pylint pytest flake8 \
+  requests \
+  jinja2 \
+  aiohttp aiodns aiohttp_jinja2 uvloop>=0.12 \
```

Comment:
Can you add `asyncinit`? https://github.com/kchmck/pyasyncinit I'm going to use this for the MySQL PR.

Reply:
Done.